### PR TITLE
chore: update documentation and schematics for webcomponents polyfill

### LINKED ISSUE #4032: Update documentation on how to include webcomponents polyfill
## Describe the bug

According to the [documentation](https://v2.clarity.design/get-started), the webcomponents polyfill should be included in `angular.json` like this:

```
"scripts": [
  ... any existing scripts
  "../node_modules/@webcomponents/custom-elements/custom-elements.min.js",
  "../node_modules/@clr/icons/clr-icons.min.js"
]
```

When building like that, I was getting an error in IE11: 

```
customElements is undefined
```

## How to fix

I fixed the problem by instead including the webcomponents polyfill in `polyfills.ts` at the very end of the file:

```
... other polyfills
import '@webcomponents/custom-elements';
```

This is also the way I found that the polyfills are loaded in the stackblitz examples. If this is indeed the way to go and this was not some other configuration issue in my project, the documentation should be updated accordingly.

## Versions

**App**

* Angular: 8
* Clarity: 2

**Device:**

* Browser IE11

### PR DESCRIPTION
**Pull request by gnomeontherun**
fixes #4032

## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [x] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #4032

## Does this PR introduce a breaking change?

* [ ] Yes
* [ ] No

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ src/website/src/sitemap.xml
 src/clr-core/**/*.css.ts
 src/clr-angular/utils/_mixins.clarity.scss
 docs/core
+**/node_modules/
+src/schematics/
```
Comment (hippee-lee) [inline .prettierignore]: Shouldn't we be formatting these files?

File: src/website/src/app/documentation/get-started/get-started.component.ts [hunk 2]
```diff
@@ -30,7 +30,8 @@ export class AppModule { }
 const HTML_IMPORTS = `
 <link rel="stylesheet" href="path/to/node_modules/@clr/ui/clr-ui.min.css">
 <link rel="stylesheet" href="path/to/node_modules/@clr/icons/clr-icons.min.css">
-<script src="path/to/node_modules/@webcomponents/custom-elements/custom-elements.min.js"></script>
+<script src="path/to/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
```
Comment (Shijir) [inline src/website/src/app/documentation/get-started/get-started.component.ts]: I wasn't aware of these dependency changes. Do people need both of these dependencies in all cases?

Comment (gnomeontherun) [reply src/website/src/app/documentation/get-started/get-started.component.ts]: This is the recommendation from @coryrylan that will be used with Core, so yes I think so.

Comment (gnomeontherun) [reply src/website/src/app/documentation/get-started/get-started.component.ts]: Also, the schematic will fix this for anyone automatically, and updated the ng-add schematic too.

File: .prettierignore (hunk 1 above)
Comment (gnomeontherun) [reply .prettierignore]: I'm going to revert this.

**Pull request merged**